app/services/create_org_and_admin_service.rb

```diff
@@ -0,0 +1,102 @@
+# class to create a new organization, user, staff account with role admin, and send email via the console
```

Comment:
We may find ourselves creating more services like this. I think renaming this to be Organizations::CreateService would start allowing us to namepace these services to keep things organized.